tests/model_forms/tests.py

```diff
@@ -270,6 +273,21 @@ def test_save_blank_false_with_required_false(self):
         obj = form.save()
         self.assertEqual(obj.name, '')
 
+    @skipIfDBFeature('interprets_empty_strings_as_nulls')
```

Comment:
Instead of skipping the test, how about modifying the expected value for the assertions based on the feature flag?